bot/utils/channel.py

```diff
@@ -46,6 +46,13 @@ def is_in_category(channel: discord.TextChannel, category_id: int) -> bool:
     return getattr(channel, "category_id", None) == category_id
 
 
+def is_voice_channel(
+        channel: discord.TextChannel | discord.DMChannel | discord.Thread | discord.StageChannel
+) -> bool:
+    """Return True if `channel` is a voice one."""
+    return channel.type in [discord.ChannelType.voice, discord.ChannelType.stage_voice]
```

Comment:
We might want to account for null channels that we ignore here.
In that case, I'm not sure what value to return 🤔

Reply:
I guess we can just let it throw, and handle those case by case if they will ever occur.